### PR TITLE
Unify styling across mobile chpt 1 & 2

### PR DESCRIPTION
**Pull request by ellenbechtel**
Before making a pull request
----------------------------
First . . .
- [x] Clean the code the way Vue likes it - run 'npm run lint --fix'        
- [ ] Make sure all tests run

Then check for accessibly compliance
- [ ] Run WAVE plugin 508 compliance tool

Then run Browserstack; check that application works on . . .
- [x] Chrome
- [x] Safari
- [ ] Edge
- [x] Firefox
- [ ] Samsung Internet
- [ ] Internet Explorer 11 (not supported, but still needs at least a working user redirect page)

Finally . . .
- [ ] Update the changelog appropriately

Title
-----------
Brief description of changes. Reference the JIRA ticket if appropriate

Description
-----------
If no ticket is referenced, describe the changes made. Note anything that you want the reviewers to know while
reviewing your pull request

After making a pull request
---------------------------
- [ ] If appropriate, put the link to the PR in the ticket
- [ ] Assign someone to review unless the change is trivial

### CODE REVIEW

File: src/components/2-Monitoring-Mobile/MatrixAvailability.vue [hunk 1]
```diff
@@ -1,5 +1,8 @@
 <template>
   <div id="matrix-availability">
+    <h3 class="viz-title">
+      Observation Frequency for Reaches in the DRB
```
Comment (cnell-usgs) [inline src/components/2-Monitoring-Mobile/MatrixAvailability.vue:4]: Should include what is being observed?

File: src/components/2-Monitoring-Mobile/MatrixExplainerColor.vue [hunk 2]
```diff
@@ -2610,328 +2552,36 @@
...
-          class="cls-84"
-          x="136.3"
-          y="0"
-        >e</tspan><tspan
-          class="cls-85"
-          x="148.9"
-          y="0"
-        >ach </tspan><tspan
-          class="cls-86"
-          x="193.2"
```
Comment (cnell-usgs) [inline src/components/2-Monitoring-Mobile/MatrixExplainerColor.vue:2639]: LOL truly hate that `ai` outputs like this

File: src/views/3-Modeling/Modeling.vue [hunk 3]
```diff
@@ -9,7 +9,7 @@
       <div class="sticky">
         <div id="bees-container">
           <div id="progress-container">
-          <p class="progress"> </p>
+            <p class="progress" />
```
Comment (cnell-usgs) [inline src/views/3-Modeling/Modeling.vue:12]: not really related to this PR but I was having issues with this `<p>` and it wanting to have a super short line height with a white background. my normal fixes did not seem to work?

**Review by cnell-usgs — APPROVED** [review]

**Review by hcorson-dosch-usgs — APPROVED** [review]
Nice, Ellen! Thanks for wrangling all of this css. It's really looking good.

Just going to note a few styling things as I notice them. Feel free to change these now or put them in an issue to be addressed later.

At some point we should update the color of the icons and associated text in ch1 (and those we add in ch2) to #6399ba, the same color as the rivers.

The 'algal' in Emergence of harmful algal blooms seems to run off screen on mobile?

This is pretty random, and is fine to leave as is if it's not a quick fix, but I noticed this upper stream segment appears thicker on mobile than on desktop?
image

I think the body text should be quite a bit smaller, both to distinguish it from the header text and to allow for a bit more negative space on the screen. Things feel a little crowded now (I'm having the same issues in the desktop ch2). I think it would also be good to have the body text match that in the svgs we inserted, so that the body text and our description of the matrix axes feel continuous. So matching the size and color of those two types of text would be great. And if the body text is grey, then our use of white to highlight important words/phrases will really stand out.

I think having some sort of header is a good idea. I think my issue with what we have currently is that we have two different header styles - the larger white font and the shadow-boxed white headers. So I think only using one of those two headers styles throughout would be good.

In the svg that explains the colors in the matrix, the 'in the dark' text looks super great now, but it is slightly overlapping the surrounding text.

I don't know that we need the header 'Observation Frequency for Reaches in the DRB'. I think the final availability matrix still fits within the previous section, which was titled 'Data Availability'. And I think this is the one spot where having another box in the midst of the content gets a bit busy visually.

The final data availability matrix box is slightly narrower than the other boxes that come before, where the matrix is being explained. I think we should either make the preceding boxes narrower, or remove the N-S axis on the final matrix so that it can be wider (though I think you were right to add it for clarity).

Also - the N-S axis line next to the availability matrix is a little hard to see, so it would be good to make that lighter if we keep it.

**Comment by hcorson-dosch-usgs** [comment]
Looking at the older version on mobile, I think that N-S arrow you added to the final matrix does look good and is helpful

**Comment by ellenbechtel** [comment]
Thanks guys!  Hayley, I'm going to copy all your edits over to Issue #58 so we can return to them all at once. 

The quick answer to the thicker stream segment - I'm not sure why that segment in particular is thicker, but a trick that I had to do for this is to convert all the strokes of different thicknesses into outlines, then merge all the outlines into one polygon.  Otherwise, it was showing up with weird jaunty lines that looked more like motherboard circuitry than a river network.  So it's probably a consequence of that!

**Comment by hcorson-dosch-usgs** [comment]
Oh that's weird. Ok - I don't think the thicker stroke matters too much, so we can put that on the back burner

**Pull request merged**